src/app/certificate/certificate.component.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { LanguageService } from '../services/language.service';
+import { APIService } from '../services/api.service';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { NavbarService } from '../services/navbar.service';
+import { MatDialogModule } from '@angular/material';
+import { CertificateService } from './certificate.component';
+import { MAT_DIALOG_DATA } from '@angular/material';
+import { CertificateComponent } from './certificate.component';
+import { MatDialogRef } from '@angular/material';
+
+
+describe('CertificateComponent', () => {
+  let component: CertificateComponent;
+  let fixture: ComponentFixture<CertificateComponent>;
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [CertificateComponent],
+      imports: [HttpClientTestingModule, MatDialogModule],
+      providers: [NavbarService, CertificateService, {
+        provide: MAT_DIALOG_DATA, useValue: {name: 'testName'}
+      },
+        APIService, {provide: MatDialogRef, useValue: {}}, LanguageService]
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CertificateComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+
+  });
+
+  it('should create', () => {
+    component.ngOnInit();
+    expect(component).toBeTruthy();
+  });
+  it('should have correct injected userName in certificate', () => {
+    expect(component.data.name).toEqual('testName');
+  })
+});
```

Comment:
new line..

Reply:
@shreyans29 I checked it. It has a new line already. Please check again.

Reply:
I think github skips it sometimes while displaying..